Source/Core/DolphinQt/Debugger/WatchWidget.cpp

```diff
@@ -132,7 +132,8 @@ void WatchWidget::Update()
        // i18n: The base 10 numeral system. Not related to non-integer numbers
        tr("Decimal"),
        // i18n: Data type used in computing
-       tr("String")});
+       tr("String"),
+       tr("Float")});
```

Comment:
This should probably have an `i18n` comment about floating point numbers (data type)

Reply:
Added.